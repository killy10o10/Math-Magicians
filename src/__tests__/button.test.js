import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../components/button';

describe('Testing if the buttons are displayed', () => {
  it('button is rendered', () => {
    const button = renderer
      .create(<Button />)
      .toJSON();
    expect(button).toMatchSnapshot();
  });
});
