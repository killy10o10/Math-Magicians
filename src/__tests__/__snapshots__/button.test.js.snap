// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing if the buttons are displayed button is rendered 1`] = `
<div
  className="btn-div"
>
  <button
    className="btn"
    type="button"
  />
</div>
`;
